test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -69,7 +69,7 @@ public void WhenGettingSystemHistory_GivenAValueForSince_TheServerShouldReturnOn
             var sinceUriString = HttpUtility.UrlEncode(since.ToString("o"));
 
             Thread.Sleep(500);  // put a small gap between since and the first edits
-            _createdResource.Resource.Comment = "Changed by E2E test";
+            ////_createdResource.Resource.Comment = "Changed by E2E test";
```

Comment:
I think these could be anything. Doesn't have to be comment if comment was removed.